CMakeLists.txt

```diff
@@ -618,6 +620,14 @@ else()
 endif()
 
 
+# Source Groups
+source_group( "Libraries" REGULAR_EXPRESSION "libs/" )
+source_group( "AirWindows" REGULAR_EXPRESSION "libs/airwindows/" )
```

Comment:
Should this be "Airwindows"?

Reply:
Yeah good point there!

Reply:
Yeah sorry we got the rest of em. Let me add a change to your def and push it over in this PR evil.

Reply:
OK forced that. Merging now since the code all built on the prior branch.